encrypted-media-respec.html

```diff
@@ -1581,15 +1581,8 @@
         The <a def-id="monitor-cdm-algorithm"></a> algorithm MUST be run in parallel to the main event loop but not in parallel to other procedures defined
         in this specification that are also defined to be run in parallel.
       </p>
-
-      <p>
-        If a <a>MediaKeySession</a> object becomes inaccessible to the page
-        and is not <a def-id="media-key-session-closed"></a>, the User Agent
-        MUST use the CDM to close the session before User Agent state
-        associated with the <a href="#key-session">session</a> is deleted.
-      </p>
       <p>
-        A MediaKeySession object SHALL NOT be destroyed and SHALL continue to receive events if it is
+        A <a>MediaKeySession</a> object SHALL NOT be destroyed and SHALL continue to receive events if it is
         not <a def-id="media-key-session-closed"></a> and the <a>MediaKeys</a> object that created it remains accessible.
         Otherwise, a MediaKeySession object that is no longer accessible to JavaScript SHALL NOT receive further
         events and MAY be destroyed.
```

Comment:
Interestingly, the spec does not require a MediaKeySession object to be destroyed when it is inaccessible. Maybe this is intended to allow GC to defer it (or for GC to never be activated if it is not necessary).